tesseract/tesseract_planning/tesseract_motion_planners/src/ompl/profile/ompl_default_plan_profile.cpp

```diff
@@ -98,15 +145,80 @@ void OMPLDefaultPlanProfile::apply(OMPLProblem& prob,
       goal_state[i] = joint_waypoint[i];
 
     prob.simple_setup->setGoalState(goal_state);
+  }
+}
 
-    // Setup state checking functionality
-    ompl::base::StateValidityCheckerPtr svc_without_collision = processStateValidator(prob, env, prob.manip_fwd_kin);
+void OMPLDefaultPlanProfile::applyStartStates(OMPLProblem& prob,
+                                              const Eigen::Isometry3d& cartesian_waypoint,
+                                              const PlanInstruction& parent_instruction,
+                                              const std::vector<std::string>& /*active_links*/,
+                                              int /*index*/)
+{
+  const auto dof = prob.manip_fwd_kin->numJoints();
```

Comment:
Would there ever be an error case where the fwd kin has a different DOF than the IK? If so, this would break `num_solutions = joint_solutions.size() / dof;`